src/Microsoft.Health.Dicom.Web/Microsoft.Health.Dicom.Web.csproj

```diff
@@ -32,4 +32,10 @@
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
+  <Target Name="SwaggerPostBuildTarget" AfterTargets="Build" Condition="'$(ContinuousIntegrationBuild)' != 'true'">
+    <Exec Command="dotnet tool restore"></Exec>
```

Comment:
If we're going to use restore, then we can skip any description in a README. Let's instead commit a tool "manifest" file and specify it here in the `dotnet tool restore` command.

Reply:
This way we can standardize our tooling across msbuild, docker, and the CI script